<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>W3 Mod Merger settings:</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icons/w3.ico</normaloff>:/images/icons/w3.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutMain">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutTop">
       <item>
        <widget class="QGroupBox" name="groupBoxWcc">
         <property name="title">
          <string>Path to wcc_lite.exe:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditWcc">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonWcc">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxModName">
         <property name="maximumSize">
          <size>
           <width>153</width>
           <height>53</height>
          </size>
         </property>
         <property name="title">
          <string>&amp;Merged mod name:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEditModName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutDirs">
       <item>
        <widget class="QGroupBox" name="groupBoxFolders">
         <property name="title">
          <string>Working folders:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelUncooked">
            <property name="text">
             <string>&amp;Uncooked:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditUncooked</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditUncooked">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonUncooked">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCooked">
            <property name="text">
             <string>Coo&amp;ked:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditCooked</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditCooked">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonCooked">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPacked">
            <property name="text">
             <string>&amp;Packed:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditPacked</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditPacked">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="buttonPacked">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBoxMisc">
           <property name="title">
            <string>Misc options:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="checkBoxAutoInstall">
              <property name="text">
               <string>&amp;Auto install merged mod</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxCleanDirs">
              <property name="text">
               <string>Auto clean working &amp;folders</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxAdvanced">
       <property name="title">
        <string>wcc options:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelUncooking">
              <property name="text">
               <string>Uncooking options:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxSkipErrors">
              <property name="text">
               <string>&amp;Skip errors</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxSwf">
              <property name="text">
               <string>Dump sw&amp;f</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineCmdUncook">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelCooking">
            <property name="text">
             <string>Cooking options:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineCmdCook"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="labelOptCache">
            <property name="text">
             <string>Build cache:</string>
            </property>
            <property name="buddy">
             <cstring>lineCmdCache</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineCmdCache">
            <property name="minimumSize">
             <size>
              <width>650</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelOptPack">
            <property name="text">
             <string>Packing:</string>
            </property>
            <property name="buddy">
             <cstring>lineCmdPack</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelOptMeta">
            <property name="text">
             <string>Metadata:</string>
            </property>
            <property name="buddy">
             <cstring>lineCmdMetadata</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineCmdPack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineCmdMetadata">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutButtons">
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>108</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditWcc</tabstop>
  <tabstop>buttonWcc</tabstop>
  <tabstop>lineEditUncooked</tabstop>
  <tabstop>buttonUncooked</tabstop>
  <tabstop>lineEditCooked</tabstop>
  <tabstop>buttonCooked</tabstop>
  <tabstop>lineEditModName</tabstop>
  <tabstop>groupBoxAdvanced</tabstop>
  <tabstop>lineCmdCache</tabstop>
  <tabstop>lineCmdPack</tabstop>
  <tabstop>lineCmdMetadata</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonReset</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
